### ISSUE TITLE
its more of recommendation 

### ISSUE THREAD

**Issue by laithalshabi** [FaridSafi/react-native-gifted-chat#2371]
#### Issue Description

the flatlist makes alot of lag and performance issue 

#### Recommendation 

i recommend to use Flashlist / Biglist because they  are much faster and better than flatlist

**Comment by fukemy** [comment]
flatlist automatic populated children height, while FlashList/Biglist need give it before render

**Comment by JoniVR** [comment]
I disagree with this, let's not depend too much on third party components that might not be maintained in the future. It would however be an option to provide the user the ability to use their own scrollview (this allows for flashlist) but I can imagine this would introduce a lot of added complexity.

**Comment by kesha-antonov** [comment]
> I disagree with this, let's not depend too much on third party components that might not be maintained in the future. It would however be an option to provide the user the ability to use their own scrollview (this allows for flashlist) but I can imagine this would introduce a lot of added complexity.

+- 1 dep is not a big deal )
and this package itself is not actively maintained also
there're 31 PR and maintainer doesn't look at them
